Lib/test/libregrtest/logger.py

```diff
@@ -32,9 +32,9 @@ def log(self, line: str = '') -> None:
 
         mins, secs = divmod(int(test_time), 60)
         hours, mins = divmod(mins, 60)
-        test_time = "%d:%02d:%02d" % (hours, mins, secs)
+        formatted_test_time = "%d:%02d:%02d" % (hours, mins, secs)
 
-        line = f"{test_time} {line}"
+        line = f"{formatted_test_time} {line}"
```

Comment:
This variable has been renamed here because there's already a variable earlier in this function called `test_time`, and it has type `float`. Mypy enforces that a variable should only ever have one consistent type within any given scope, and here the name is reassigned to a `str`. We can fix the mypy complaints by using a new variable name for the `str` value, rather than reusing the `test_time` name

Reply:
test_time is a bad name, in fact, it's just time relative to regrtest start time.

I suggest renaming test_time to log_time and the second variable to formatted_log_time.

Reply:
Good shout, I'll do the renames

Reply:
In the past, I put "test" *everywhere* :-) "test" now means nothing, since everything is called "test" in libregrtest :-) That's why I renamed "runtest.py" to "single.py" and rename "runtest_mp.py" to "run_worker.py" ;-)